usermods/Tubes/pattern.h

```diff
@@ -65,8 +66,8 @@ void biwave(VirtualStrip *strip)
   uint16_t r = strip->frame * 32;
   r = cos16( r + random_offset ) + 32768;
 
-  uint8_t p1 = scaled16to8(l, 0, strip->length()-1);
-  uint8_t p2 = scaled16to8(r, 0, strip->length()-1);
+  uint8_t p1 = std::min(scaled16to8(l, 0, strip->length()), (uint8_t)(strip->length()-1));
```

Comment:
not sure this fixed anyting, but was wondering if because of integer rounding, the last pixel wasn't seeing the effects as often